lib/pub_sub/message.rb

```diff
@@ -11,20 +11,22 @@ def initialize(payload)
     end
 
     def process
-      validate_message!
+      begin
+        validate_message!
+      rescue PubSub::ServiceUnknown => e
```

Comment:
Sure, but at https://github.com/westfieldlabs/pub_sub/blob/2002d999a6f541a2542501a377d37647bf2bb4ff/lib/pub_sub/tasks/pub_sub.rake#L17 there's a rescue for this exact case.

I don't think it makes sense to have both of these rescues (where exactly the rescue belongs I don't have a strong opinion on).